agenttest.py

```diff
@@ -332,21 +332,20 @@ def send_arps(hosts):
 # pylint: disable=too-many-locals, too-many-statements
 
 
-def end_to_end_test():
+def end_to_end_test(options, envs):
     """Simple end-to-end test of FAUCET config agent"""
 
-    # Start with empty FAUCET config file
-    write_file(FAUCET.cfile, '')
-
     info('\n* Generating certificates\n')
     make_certs()
     params = dict(cert_dir=CERT_DIR, gnmi_port=GNMI_PORT, cfile=FAUCET.cfile)
     client_auth = (' -ca {cert_dir}/fakeca.crt -cert {cert_dir}/fakeclient.crt'
```

Comment:
Since it's Python 3, we can use an f-string (`f'...'`) here I think instead of a params dict.